screens/HomeScreen.tsx

```diff
@@ -141,8 +144,12 @@ const HomeScreen = ({
   updateSpotPrice,
   updateTokensMeta,
   updateTransactions,
-  updateUtxos
+  updateUtxos,
+  tokenBlacklist
 }: Props) => {
+  const [isRevealed, setToggled] = useState(false);
```

Comment:
1. `isRevealed` is not descriptive, what is revealed?  On the home-screen multiple things may be revealable later on.  Rename to more descriptive, such as `isHiddenTokensRevealed` or `isShowingBlacklist`, etc etc.  

2.  Follow the useState hook naming convention where the `setter` method is named as `setVariableName`.  In this case, that would be `setIsRevealed` (changing depending on how the variable name changes)